fagiServer/src/main/java/com/fagi/model/Data.java

```diff
@@ -44,7 +44,15 @@ public synchronized Conversation createConversation(List<String> participants) {
 
         String name = participants
             .stream()
-            .reduce("", (a, b) -> a + ", " + b);
+            .reduce(
+                    "",
+                    (a, b) -> {
+                        if (!a.isEmpty()) {
+                            return a + ", " + b;
```

Comment:
What about if b is empty?

Reply:
Now uses StringJoiner to better handle the name creation.